### PR TITLE
LPS-126479

### PR DESCRIPTION
**Pull request by mlan0**
Related Issue: https://issues.liferay.com/browse/LPS-126479

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js [hunk 1]
```diff
@@ -46,14 +47,16 @@ import {paginationReducer} from './reducers/index.es';
  * the submit of the Form in the User View, removes the default behavior
  * of the submit and uses Liferay.Util.submitForm.
  */
+
+const DDM_FORM_PORTLET_NAMESPACE =
```
Comment (natocesarrego) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js:51]: @mlan0, can you please move this variable declaration to be on top of the comment above?

This comment is associated to the method `submitForm`, previously named `handleFormSubmitted`. :wink:

**Comment by natocesarrego** [comment]
Lovely pull, @mlan0! :heart_eyes: :clap:

**Comment by liferay-forms** [comment]
Hey @mlan0, can you please check if these failures are related to your changes? Thanks

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js (hunk 1 above)
Comment (mlan0) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js]: Done!

**Comment by mlan0** [comment]
ci:test:sf

**Comment by mlan0** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js [hunk 2]
```diff
@@ -97,6 +99,69 @@ const useFormSubmit = ({apiRef, containerRef}) => {
 		[apiRef, containerRef, submittable]
 	);
 
+	const handleFormSubmitted = useCallback(
+		(event) => {
+			event.preventDefault();
+
+			if (portletNamespace === DDM_FORM_PORTLET_NAMESPACE) {
```
Comment (natocesarrego) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js:106]: @mlan0, can you please send a new complementary pull replacing this code by this [code](https://github.com/natocesarrego/liferay-portal/pull/1100/commits/08c695ab9afee269c013d57e3914721c037ce384)?

This modification is necessary to allow this fix work on a Form Portlet added to a Site Page as a Widget. If you have any question, please let me know.

Thanks

Comment (mlan0) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/FormView.es.js:106]: Done!

**Pull request closed without merging**